spec/models/invenio_rdm_record_converter_spec.rb

```diff
@@ -170,19 +171,24 @@
       },
       "extras": {
         "presentation_location": generic_file.based_near,
-        "owner": {
-          "netid": user.username,
-          "email": user.email
-        },
         "permissions": {
-          "read": ["public"],
-          "edit": [user.username]
-        }
+          "owner": {
+            user.username => user.email
+          },
+          "read": {
+            "public": ""
+          },
+          "edit": {
+            user.username => user.email,
+          }
+        }.with_indifferent_access
```

Comment:
:heart: that's a nice structure. Really like the owner in permissions because ownership is about permission. Thanks for that @sharpattack